### PR TITLE
Department CRUD added.

### PR DESCRIPTION
**Pull request by deAshish**

### CODE REVIEW

**Review by NirmalSilwal — CHANGES_REQUESTED** [review]
In AmpApplication.java why have you removed model mapper part? Are you not using Model Mapper in our code base?

update: My bad! I see there was two ModelMapper definition. Ignore this comment :) 😀

**Review by NirmalSilwal — COMMENTED** [review]
Looks good to me! **update** logic needs to be reviewed again. Also if possible we can enhance **DepartmentServiceImp**, **updateById()** method implementation as in our case currently we have only one field - departmentName and we're changing each property. This could be problem when we have so many fields in the class. Not sure if there is better way to update. Will check on my end also.

Would like other reviewers to address it if they have solution in mind. Otherwise, we can merge this PR.

Thanks

File: server/src/main/java/edu/miu/amp/AmpApplication.java [hunk 1]
```diff
@@ -17,10 +17,5 @@ public ModelMapper modelMapper() {
 	}
 
 
-	@Bean
```
Comment (pravashupreti) [inline server/src/main/java/edu/miu/amp/AmpApplication.java:20]: @deAshish this bean is used on service layer to map dto to entity and viceversa

**Review by pravashupreti — COMMENTED** [review]
Removing modelmaper bean may break some apis

**Pull request merged**